server/src/test/java/io/spine/server/procman/ProcessManagerTest.java

```diff
@@ -124,7 +124,7 @@
     private TestProcessManager processManager;
 
     @BeforeEach
-    void setUp() {
+    void initProcessManager() {
```

Comment:
A better name would be `initContextAndProcessManager()`. Just to play nicely with `closeContext()`.